test/toInteger32.js

```diff
@@ -0,0 +1,74 @@
+import * as R from 'ramda';
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('toInteger32', function () {
+  context('given a small integer', function () {
+    specify('should maintain the same value', function () {
+      assert.strictEqual(RA.toInteger32(1 + Math.random()), 1);
```

Comment:
Please don't use `Math.random` in tests. Use numbers constants like `1` or `200` to make the test explicit and clear. This is not `property based test` so randomization of values in not something we want to do here.

Reply:
This applies for all the tests in this file.

Reply:
Fair enough if you don't want randomisation - I'll change it